Document Java Source Context (#7086)
getsentry/sentry-docs: src/platforms/java/common/gradle.mdx

}

plugins {
    id("io.sentry.jvm.gradle") version "{{@inject packages.version('sentry.java.android.gradle-plugin', '3.9.0') }}"
}
```

<Note>

> Although the plugin repository name contains `android`, it can be used for non-android projects. However, for non-android projects, the plugin only collects external dependencies and uploads <PlatformLink to="/source-context/">Source Context</PlatformLink>.

Reviewer: ```suggestion
Although the plugin repository name contains `android`, it can be used for non-Android projects.

For non-Android projects, the plugin only collects external dependencies and uploads <PlatformLink to="/source-context/">Source Context</PlatformLink>.
```

}
```

## Source Context

> Check out our dedicated page for <PlatformLink to="/source-context/">Source Context</PlatformLink>

Reviewer: ```suggestion
See our documentation on <PlatformLink to="/source-context/">Source Context</PlatformLink>.
```
